src/Symfony/Component/HttpKernel/Attribute/MapQueryParameter.php

```diff
@@ -24,12 +24,27 @@ final class MapQueryParameter extends ValueResolver
     /**
      * @see https://php.net/filter.filters.validate for filter, flags and options
      *
-     * @param string|null $name The name of the query parameter. If null, the name of the argument in the controller will be used.
+     * @param class-string $resolver The class name of the resolver to use
      */
     public function __construct(
+        /**
+         * The name of the query parameter. If null, the name of the argument in the controller will be used.
+         */
         public ?string $name = null,
+
+        /**
+         * The filter to pass to "filter_var()".
+         */
         public ?int $filter = null,
+
+        /**
+         * The flags to pass to "filter_var()".
```

Comment:
`@var int-mask-of<FILTER_FLAG_*|FILTER_NULL_ON_FAILURE>` to tell SA tools that this is a bitmask of the flag constants.

Reply:
Same as above : 
![image](https://github.com/symfony/symfony/assets/2144837/03e5650b-f1ef-4191-ad41-db18997d9a67)

I think we should use the notation used in ParameterBag

Reply:
Also thanks for the tip with `int-mask-of`, I wasn't aware of this notation :slightly_smiling_face:

Reply:
Same issue with global const.